app/services/decidim/verifications/sant_boi_census/sant_boi_census_authorization_service.rb

```diff
@@ -20,17 +20,19 @@ def initialize(document_type, document_number)
         #   body   => WS response body, as Nokogiri::XML instance
         #   status => WS response status, as Integer
         def perform_request
-          return @response if defined?(@response)
-
           @token = retrieve_login_response(perform_login_request.body)
+
           response = perform_do_operation_tao_request
 
-          @response = { status: response.status,
-                        body: retrieve_get_habitante_by_dni_response(response.body) }
+          {
+            status: response.status,
+            body: retrieve_get_habitante_by_dni_response(response.body)
+          }
```

Comment:
Stopped caching the response since we are not instantiating an object in `SantBoiCensusAuthorizationHandler::perform_request` anyways